dome/install_windows_src.md

```diff
@@ -0,0 +1,157 @@
+
+1. Install a [Conda package management system](https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html).
+   Miniconda suffices. You will likely want to check the box to add `conda` to your `PATH`
+   during the installation process so that you won't have to do this step manually.
+
+2. Install [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/).
+   The Community version is free for students, open-source developers, and personal
+   development. Check "Desktop development with C++" in the Workloads tab,
+   and uncheck "C++ Cmake Tools." We will install cmake via Conda.
+
+3. Open a Visual Studio Command Prompt (search for "x64 Native Tools Command Prompt
+   for VS 2019" in the Windows search field near the Windows button). Optionally,
+   right-click and pin to the task bar for quick access in the future.
+
+  If you did not add Conda to your `PATH` environment variable during Conda installation,
+  you may need to navigate to the location of `condabin` in order to use the `conda` command.
+  To find `condabin`, search for "Anaconda Prompt" in the Windows search field near the
+  Windows button, open it, run `where conda`, and look for a line containing the directory `condabin`.
+
+4. Navigate to your `condabin`, if necessary, and then create and activate a Conda environment:
+  ```bash
+  conda create -n ign-ws
+  conda activate ign-ws
+  ```
+
+  Once you have activate an environment, a prefix like `(ign-ws)` will be prepended to
+  your prompt, and you can use the `conda` command outside of the `condabin` directory.
+
+  You can use `conda info --envs` to see all of your environments.
+
+  To remove an environment, use `conda env remove --name <env_name>`.
+
+5. Install dependencies:
+
+  ```bash
+  conda install cmake git vcstool curl pkg-config ^
+  colcon-common-extensions eigen freeimage gts ^
+  glib dlfcn-win32 ffmpeg ruby tinyxml2 tinyxml ^
+  protobuf urdfdom zeromq cppzmq ogre jsoncpp ^
+  libzip qt --channel conda-forge
+  ```
+
+6. Navigate to where you would like to build the library, create and enter your workspace directory,
+   create the `src` directory which will contain the Ignition source code, and then clone the repositories.
+  ```bash
+  mkdir ign-ws
+  cd ign-ws
+  curl -O https://raw.githubusercontent.com/ignition-tooling/gazebodistro/master/collection-dome.yaml
+  mkdir src
+  vcs import src < collection-dome.yaml
+  ```
+
+**Note**
+
+You can view all available versions of a specific package with:
+```bash
+conda search libignition-<package_name>* --channel conda-forge
+```
+and view their dependencies with
+```bash
+conda search libignition-<package_name>* --channel conda-forge --info
+```
+and install a specific minor version with
+```bash
+conda install libignition-<package_name>=<major>.<minor>.<patch> --channel conda-forge
+```
+where `<major>` is the major release number, `<minor>` is the minor release number, and `<patch` is the patch release number.
+
```

Comment:
I think we don't need this tip for the source installation, since they're building everything from source. This is more for if they wanted to install some dependencies from binary and build others from source.

Reply:
Good point, removed the notes